app/infrastructure/services/otp_ticket_service.py

```
@@ -62,68 +83,53 @@ def generate_otp(self) -> str:
+                self.ATOMIC_EXCHANGE_SCRIPT,
+                3,
+                otp_key,
+                ticket_key,
+                attempts_key,
+                otp,
+                email,
+                settings.SIGNUP_TICKET_TTL_SECONDS,
+                5,
+            )
+        except ResponseError as exc:
+            raise OTPInvalidError('Invalid or expired OTP') from exc
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)